src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/reference/Reference.vue

```diff
@@ -8,7 +8,11 @@
       <v-container class="bg-white">
         <h2>{{ inviteTypeTitle }}</h2>
         <div role="doc-subtitle">{{ `For applicant: ${wizardStore.wizardData.applicantFirstName} ${wizardStore.wizardData.applicantLastName}` }}</div>
-        <v-btn v-if="wizardStore.step !== 1" variant="text" rounded="lg" color="primary" @click="handleBack">{{ "< Back" }}</v-btn>
+
+        <v-btn v-if="wizardStore.step !== 1" slim variant="text" rounded="lg" color="primary" @click="handleBack">
+          <v-icon size="x-large" icon="mdi-chevron-left" />
```

Comment:
You can use prepend-icon here:

`<v-btn prepend-icon="mdi-close"...`

Reply:
tried prepend-icon, did not align well, so went with this approach